<?xml version="1.0"?>
<launch>
  <arg name="addr" default="" />
  <arg name="use_standard_msgs" default="false" />
  <arg name="autorepeat_rate" default="0" if="$(arg use_standard_msgs)" />

  <node pkg="ds4_driver" type="ds4_driver_node.py" name="ds4_driver" output="screen" >
    <rosparam command="load" file="$(find ds4_driver)/config/params.yaml" />
    <param name="device_addr" value="$(arg addr)" />
    <param name="use_standard_msgs" value="$(arg use_standard_msgs)" />
    <param name="autorepeat_rate" value="$(arg autorepeat_rate)" if="$(arg use_standard_msgs)" />
  </node>

  <node pkg="tf2_ros" type="static_transform_publisher" name="ds4_to_imu"
    args="0 0.05 -0.01 -1.5707 0 1.5707 ds4 ds4_imu" />
</launch>
